src/common/enums/messaging.queue.ts

```diff
@@ -9,4 +9,6 @@ export enum MessagingQueue {
   SUBSCRIPTION_PROFILE_VERIFIED_CREDENTIAL = 'alkemio-subscriptions-profile-verified-credential',
   SUBSCRIPTION_CALLOUT_ASPECT_CREATED = 'alkemio-subscriptions-callout-aspect-created',
   SUBSCRIPTION_CALLOUT_MESSAGE_CREATED = 'alkemio-subscriptions-callout-message-created',
+  SUBSCRIPTION_CHALLENGE_OPPORTUNITY_CREATED = 'alkemio-subscriptions-challenge-opportunity-created',
+  SUBSCRIPTION_HUB_CHALLENGE_CREATED = 'alkemio-subscriptions-hub-challenge-created',
```

Comment:
any reason to include "challenge" and "hub" in the names?

Reply:
removing them.